pages/packages/permissions.md

```diff
@@ -0,0 +1,84 @@
+
+After selecting a team, you can view the member-, pipeline-, test suite-, registry- and team-specific settings.
+
+### Team-level permissions
+
+Learn more about what _team members_ are and what _team maintainers_ can do in the [Team-level permissions section of the Pipelines documentation](/docs/team-management/permissions#manage-teams-and-permissions-team-level-permissions).
+
+### Registry-level permissions
+
+When the [teams feature is enabled](#manage-teams-and-permissions), any user can create a new registry, as long as this user is a member of at least one team within the Buildkite organization, and this team has the **Create registries** [team member permission](/docs/team-management/permissions#manage-teams-and-permissions-team-level-permissions).
+
+When you create a new registry in Buildkite:
+
+- You are automatically granted the **Read & Write** permission to this registry.
+- Any members of teams to which you provide access to registry are also granted the **Read & Write** permission.
+
+The **Full Access** permission on a registry allows you to:
+
+- View and download packages, images, or modules from the registry.
+- Publish packages, images, or modules to the registry.
+- Edit the registry's settings.
+- Delete the registry.
+- Provide access to other users, by adding the registry to other teams that you are a [team maintainer](#manage-teams-and-permissions-team-level-permissions) on.
+
+Any user with **Full Access** permissions to a registry can change its permission to either:
+
+- **Read & Write**, which allows you to publish packages, images, or modules to the registry, as well as view and download these items from the registry, but _not_:
+    * Edit the registry's settings.
+    * Delete the registry.
+    * Provide access to other users.
+- **Read Only**, which allows you to view and download packages, images, or modules from the registry only, but _not_:
+    * Publish such items to the registry.
+    * Edit the registry's settings.
+    * Delete the registry.
+    * Provide access to other users.
+
+A user who is a member of at least one team with **Full Access** permissions to a registry can change the permissions on this registry. However, once this user loses this **Full Access** through their last team with access to this registry, the user then loses the ability to change the registry's permissions.
+
+Another user with **Full Access** to this registry or a [Buildkite organization administrator](#manage-teams-and-permissions-organization-level-permissions) is required to change the registry's permissions back to **Full Access** again.
+
+## Manage organization security for registries
+
+Enterprise customers can configure select registry action permissions for all users across their Buildkite organization. These features can be used either with or without the [teams feature enabled](#manage-teams-and-permissions).
```

Comment:
I was trying to re-write what we had previously (for pipelines)...

> Enterprise customers can control user permissions for selected pipeline actions.

Where 'selected' or 'select' (in the re-written version) refers to a subset of registry action permissions (since there might actually be more).

Nevertheless, I can see how 'select' might be confused with the verb/action to 'click' something (which I personally think is a better verb to use when it comes to interacting with a single 'optionless' object in the UI, as opposed to the Microsoft style guidance on using the one verb 'select' for all objects one can interact with in the UI).

Anyway, I digress... 'select' in this context doesn't add anything and so I'll accept your suggestion here.